client/volume.go

```diff
@@ -132,7 +132,7 @@ func (v *VolumeMgr) ExtendVolume(volID string, body ExtendVolumeBuilder) (*model
 	var res model.VolumeSpec
 	url := strings.Join([]string{
 		v.Endpoint,
-		urls.GenerateNewVolumeURL(urls.Client, v.TenantId, volID+"/action")}, "/")
+		urls.GenerateNewVolumeURL(urls.Client, v.TenantId, volID+"/resize")}, "/")
```

Comment:
I would prefer `urls.GenerateNewVolumeURL(urls.Client, v.TenantId, volID, "resize")}, "/")`, and BTW it could be better if define resize as a constants. For example, just define `ResizeAction` as "resize".